lib/cfg/cfg-parsers/llvm-ir.ts

```diff
@@ -46,9 +46,9 @@ export class LlvmIrCfgParser extends BaseCFGParser {
 
     constructor(instructionSetInfo: BaseInstructionSetInfo) {
         super(instructionSetInfo);
-        this.functionDefinition = /^define .+ @(.+)\([^(]+$/;
```

Comment:
This fails eg for
```
define internal fastcc void @"_ZN114_$LT$core..iter..adapters..flatten..FlatMap$LT$I$C$U$C$F$GT$$u20$as$u20$core..iter..traits..iterator..Iterator$GT$4next17h6bfdef8d5f43e5cfE"(ptr noalias nocapture noundef writeonly align 8 dereferenceable(24) %_0, ptr noalias noundef align 8 dereferenceable(128) %self) unnamed_addr personality ptr @rust_eh_personality {
```
And sadly the same regex also needs to work on the demangled version which contains not just quotes, but *spaces*:
```
define internal fastcc void @"<core::iter::adapters::flatten::FlatMap<I,U,F> as core::iter::traits::iterator::Iterator>::next"(ptr noalias nocapture noundef writeonly align 8 dereferenceable(24) %_0, ptr noalias noundef align 8 dereferenceable(128) %self) unnamed_addr personality ptr @rust_eh_personality {
```